app/main/views/briefs.py

```diff
@@ -136,23 +136,31 @@ def list_briefs():
             status.strip() for status in request.args['status'].split(',')
             ))
 
+    meta = {
+        "total": briefs.count(),
```

Comment:
Pagination objects have a [`.total` attribute](http://flask-sqlalchemy.pocoo.org/2.1/api/#flask.ext.sqlalchemy.Pagination.total), so I think we can avoid doing a separate count.